builtin/game/item.lua

```diff
@@ -446,12 +446,19 @@ function core.item_drop(itemstack, dropper, pos)
 	local obj = core.add_item(p, item)
 	if obj then
 		if dropper_is_player then
+			local vel = dropper:get_player_velocity()
 			local dir = dropper:get_look_dir()
-			dir.x = dir.x * 2.9
-			dir.y = dir.y * 2.9 + 2
-			dir.z = dir.z * 2.9
+			dir.x = vel.x + dir.x * 4
+			dir.y = vel.y + dir.y * 4 + 2
+			dir.z = vel.z + dir.z * 4
```

Comment:
The previous `* 2.9` was actually carefully tuned for the dropped item to not land too close or too far away. I prefer we stay with `* 2.9`.

Reply:
See https://github.com/minetest/minetest/commit/604fe2083d70fea92b6c53c9a9d1c1ffdcb36610